src/Components/ProductCard/ProductCard.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react'
+import { useNavigate } from 'react-router';
+import { useAuth, useCart } from '../../Context/context'
+import './ProductCard.css'
+import notifyToast from '../Toast/notifyToast';
+import { searchWishList, serverAddToWishlist, serverRemoveFromWishlist } from '../../utils/wishlist.util';
+
+function ProductCard({ product, setAuthModal }) {
+
+    const navigate = useNavigate();
+    const { cartState, cartDispatch } = useCart();
+    const { authState: { isUserLoggedIn, currentUserId } } = useAuth();
+    
+    function wishListToggle(e) {
+        e.stopPropagation()
+        if (isUserLoggedIn) {
+            if (searchWishList(cartState, product) === true) {
```

Comment:
Can avoid comparing the condition with `true` since the function returns boolean value